test/unit/util.test.ts

```diff
@@ -59,12 +59,12 @@ test('Get full URL', async(t) => {
             'https://bm.wikipedia.org/w/resources/src/mediawiki.skinning/images/spinner.gif?ca65b',
             'Full URL for image');
 
-    t.equal(getFullUrl('./-/j/js_modules/jsConfigVars.js', new URL('https://bm.wikipedia.org/')),
-            'https://bm.wikipedia.org/-/j/js_modules/jsConfigVars.js',
+    t.equal(getFullUrl('./-/sources/jsConfigVars.js', new URL('https://bm.wikipedia.org/')),
+            'https://bm.wikipedia.org/-/sources/jsConfigVars.js',
             'Full Url for relative path with skipping one file');
 
-    t.equal(getFullUrl('../-/j/js_modules/jsConfigVars.js', 'https://bm.wikipedia.org/'),
-            'https://bm.wikipedia.org/-/j/js_modules/jsConfigVars.js',
+    t.equal(getFullUrl('../-/sources/jsConfigVars.js', 'https://bm.wikipedia.org/'),
+            'https://bm.wikipedia.org/-/sources/jsConfigVars.js',
```

Comment:
Where is this « sources » coming from?

Reply:
might have been caused after rebasing extremely sorry